### PR TITLE
Drop deprecated web library

### PR DESCRIPTION
**Pull request by kevmoo**
Fixes https://github.com/flutter/flutter/issues/148219

override reason: There is no user benefit for publishing this now.

### CODE REVIEW

File: packages/image_picker/image_picker_for_web/example/integration_test/image_resizer_test.dart [hunk 1]
```diff
@@ -136,5 +135,5 @@ web.Blob _base64ToBlob(String data) {
     n--;
   }
 
-  return Blob(<JSUint8Array>[u8arr.toJS].toJS);
+  return web.Blob(<JSUint8Array>[u8arr.toJS].toJS);
```
Comment (stuartmorgan) [inline packages/image_picker/image_picker_for_web/example/integration_test/image_resizer_test.dart:138]: It looks like this is in `0.5.1`; do we actually need to require 1.0.0 to make this change?

File: packages/video_player/video_player_web/pubspec.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ dependencies:
   flutter_web_plugins:
     sdk: flutter
   video_player_platform_interface: ^6.2.0
-  web: ">=0.5.1 <2.0.0"
+  web: ^1.0.0
```
Comment (stuartmorgan) [inline packages/video_player/video_player_web/pubspec.yaml:25]: If `web/helpers.dart` was already unused, we shouldn't need to make this change.

Comment (kevmoo) [reply packages/video_player/video_player_web/pubspec.yaml:25]: ack

**Comment by kevmoo** [comment]
@stuartmorgan – PTAL

File: packages/image_picker/image_picker_for_web/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
+## NEXT
```
Comment (stuartmorgan) [inline packages/image_picker/image_picker_for_web/CHANGELOG.md:1]: This should be reverted

File: packages/image_picker/image_picker_for_web/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,5 @@
...
+
```
Comment (kevmoo) [inline packages/image_picker/image_picker_for_web/CHANGELOG.md:2]:
```suggestion
```

**Review by stuartmorgan — APPROVED** [review]
LGTM!

**Pull request merged**